embed_and_reduce/emb_and_reduce.py

```diff
@@ -100,7 +99,7 @@ def embed_and_reduce(self, search_string):
                     og tjenesteydelser, der bogføres på hovedkontiene 0-6, bortset fra momsbetalinger, 
                     der kan fradrages som indgående moms i et momsregnskab. Stk. 2. Beløbene efter stk. 
                     1 bestemmes som summen af følgende udgifter til moms (1+2): """
-
-    instance = EmbAndReduce()
+    ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+    instance = EmbAndReduce(Path(ROOT_DIR).parent / "pca.pkl", model_path=Path.home() / "bert")
```

Comment:
Jeg er også lidt i tvivl omkring, hvad vi gerne vil opnå med PathLib i stedet for bare at bruge OS.
Skal vi kunne køre python filen andre steder fra end fra base path til lib?
Vi kan godt lave en glob, der søger pakken igennem efter de relevante filer, hvis det er.
I så fald vil vi kunne køre pakken, fra alle dirs